incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/view/PathView.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.binding.openapi.internal.view;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import io.aklivity.zilla.runtime.binding.openapi.internal.model.Operation;
+import io.aklivity.zilla.runtime.binding.openapi.internal.model.PathItem;
+
+public class PathView
+{
+    private final Map<String, Operation> methods;
+
+    public PathView(
```

Comment:
For consistency with other classes, this constructor should be `private`, agree?

Please move all the private constructors / methods to the end of the class for better readability.